lib/dash/mpd_utils.js

```diff
@@ -71,11 +71,13 @@ shaka.dash.MpdUtils = class {
       }
 
       if (name == 'Time') {
-        goog.asserts.assert(typeof value == 'number',
-            'Time value should be a number!');
-        goog.asserts.assert(Math.abs(value - Math.round(value)) < 0.2,
-            'Calculated $Time$ values must be close to integers');
-        value = Math.round(value);
+        if (typeof value != 'bigint') {
+          goog.asserts.assert(typeof value == 'number',
+              'Time value should be a number or bigint!');
+          goog.asserts.assert(Math.abs(value - Math.round(value)) < 0.2,
```

Comment:
Perhaps this could be converted to a shaka.log.alwaysWarn.  Assertions are removed in production builds, but this is a condition triggered purely by input we don't control, and the app developer should probably know.

Reply:
Math is not working with bigint as far I know

Reply:
That's fine.  I'm just saying in the non-bigint code path, you should check if abs(diff) < 0.2, and alwaysWarn if not.

Reply:
Done!